www/webapp/main.py

```diff
@@ -18,3 +18,11 @@
 @webapp.route('/')
 def index():
     return render_template("index.html")
+
+@webapp.route('/register')
+def register():
+    return render_template("register.html")
+
+@webapp.route('/login')
+def login():
+    return render_template("login.html")
```

Comment:
I don't think this is where they should go, we can create a Blueprint without prexif so they attach to the base route without having them here

Reply:
Sounds good, I will add that change in another pr